pom.xml

```diff
@@ -75,6 +75,7 @@
     <dependency>
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-web</artifactId>
+      <version>3.1.0</version>
```

Comment:
@vishalsingh2972 never add versions to dependencies, which mansged by spring boot itself